docs/docsite/rst/porting_guides/porting_guide_2.7.rst

```diff
@@ -71,6 +71,25 @@ In Ansible 2.7 a new module argument named ``public`` was added to the ``include
 
 There is an important difference in the way that ``include_role`` (dynamic) will expose the role's variables, as opposed to ``import_role`` (static). ``import_role`` is a pre-processor, and the ``defaults`` and ``vars`` are evaluated at playbook parsing, making the variables available to tasks and roles listed at any point in the play. ``include_role`` is a conditional task, and the ``defaults`` and ``vars`` are evaluated at execution time, making the variables available to tasks and roles listed *after* the ``include_role`` task.
 
+include_tasks/import_tasks inline variables
+```````````````````````````````````````````
```

Comment:
I can't tell whether you meant this to be a sub-section - if so, it's fine. If not, please make the heading match the others at this level (---------- instead of ``````````).

Reply:
Thanks for the update. Looks good, merging.